src/components/ReportViewer/log-table.js

```diff
@@ -84,15 +86,36 @@ const LogTable = ({ logs }) => {
                 severity: log.getSeverityName(),
                 message: log.getLog(),
                 backgroundColor: log.getColorName(),
+                reportId: log.getReportId(),
             };
         });
     };
 
+    const handleRowClick = (data) => {
+        setSelectedRowIndex(data.index);
+        onRowClick(data);
+    };
+
+    const rowStyleFormat = (row) => {
+        if (row.index < 0) return;
+        if (selectedRowIndex === row.index) {
+            return {
+                backgroundColor: 'lightblue',
```

Comment:
Does it play well with both dark and light themes ?

Reply:
the answer is no: 
![Capture d’écran de 2022-11-10 10-35-04](https://user-images.githubusercontent.com/89208/201054543-41b0ca79-1112-44e7-883c-d8a4c5d0d6d6.png)

Reply:
Modified to use colors defined for selected rows in themes.